### ISSUE TITLE
Accessibility.speak should be cancelable

### ISSUE THREAD

**Issue by danielsogl** [ionic-team/capacitor#1979]
Description of the problem:
It should be possible to cancel the speak function otherwise the user has to wait until the text has been finished before he can listen to another text.

Affected platform
- [x] Android
- [x] iOS
- [x] electron
- [x] web

OS of the development machine
- [ ] Windows
- [x] macOS
- [ ] linux

Other information:

Capacitor version:

node version:

npm version:

CocoaPods version:

Steps to reproduce:

Link to sample project:

**Comment by jcesarmobile** [comment]
Issues tagged with feature request are closed but tracked for reactions to gauge interest.

But by a quick check at the Plugin code, this doesn't seem possible on iOS. While on Android, it seems to work that way already, if a new text is sent the old one should be flushed. 
Should be possible on web
